run/john.conf

```diff
@@ -106,6 +106,15 @@ LogDateStderrFormat =
 # parsed as ESC for use in ANSI codes (like in the default)
 TerminalReset = ^[0m
 
+# If this is true, colorize some errors/warnings - such as test failures.
+# The color codes can be redefined here if needed. Any "^" characters will be
+# parsed as ESC for use in ANSI escape sequences (like in the defaults here,
+# which define red for errors and yellow for warnings.
+UseColors = Y
+ColorError = ^[0;31m
+ColorWarning = ^[0;33m
```

Comment:
It's a bit weird to define `ColorWarning` without introducing _any_ use of it soon. Maybe it's better to go over the places where we already use colors and add `Color*` categories for those?